src/EnergyPlus/HVACManager.cc

```diff
@@ -2534,225 +2397,222 @@ void ReportAirHeatBalance(EnergyPlusData &state)
-                    ZnAirRpt(ZoneLoop).VentilTotalGain = TotalLoad * ADSCorrectionFactor;
-                    ZnAirRpt(ZoneLoop).VentilTotalLoss = 0.0;
+                    znAirRpt.VentilTotalGain = TotalLoad * ADSCorrectionFactor;
+                    znAirRpt.VentilTotalLoss = 0.0;
                 } else {
-                    ZnAirRpt(ZoneLoop).VentilTotalGain = 0.0;
-                    ZnAirRpt(ZoneLoop).VentilTotalLoss = -TotalLoad * ADSCorrectionFactor;
+                    znAirRpt.VentilTotalGain = 0.0;
+                    znAirRpt.VentilTotalLoss = -TotalLoad * ADSCorrectionFactor;
                 }
             }
         }
 
         if (ADSCorrectionFactor > 0 && VentZoneNum > 1 && VentZoneMassflow > 0.0) {
-            ZnAirRpt(ZoneLoop).VentilAirTemp /= VentZoneMassflow;
+            znAirRpt.VentilAirTemp /= VentZoneMassflow;
         } else if (ADSCorrectionFactor > 0 && VentZoneNum == 1) {
-            ZnAirRpt(ZoneLoop).VentilAirTemp = VentZoneAirTemp;
+            znAirRpt.VentilAirTemp = VentZoneAirTemp;
         } else { // Just in case
-            ZnAirRpt(ZoneLoop).VentilAirTemp = Zone(ZoneLoop).OutDryBulbTemp;
+            znAirRpt.VentilAirTemp = zone.OutDryBulbTemp;
         }
 
         // Report mixing sensible and latent loads
         MixSenLoad(ZoneLoop) = 0.0; // Initialize arrays to zero before starting to sum
         MixLatLoad(ZoneLoop) = 0.0;
-        ZnAirRpt(ZoneLoop).MixVolume = 0.0;         // zero reported volume prior to summations below
-        ZnAirRpt(ZoneLoop).MixVdotCurDensity = 0.0; // zero reported volume flow rate prior to summations below
-        ZnAirRpt(ZoneLoop).MixVdotStdDensity = 0.0; // zero reported volume flow rate prior to summations below
-        ZnAirRpt(ZoneLoop).MixMass = 0.0;           // ! zero reported mass prior to summations below
-        ZnAirRpt(ZoneLoop).MixMdot = 0.0;           // ! zero reported mass flow rate prior to summations below
+        znAirRpt.MixVolume = 0.0;         // zero reported volume prior to summations below
+        znAirRpt.MixVdotCurDensity = 0.0; // zero reported volume flow rate prior to summations below
+        znAirRpt.MixVdotStdDensity = 0.0; // zero reported volume flow rate prior to summations below
+        znAirRpt.MixMass = 0.0;           // ! zero reported mass prior to summations below
+        znAirRpt.MixMdot = 0.0;           // ! zero reported mass flow rate prior to summations below
         //    MixingLoad = 0.0d0
 
         for (int MixNum = 1; MixNum <= state.dataHeatBal->TotMixing; ++MixNum) {
-            if ((Mixing(MixNum).ZonePtr == ZoneLoop) && Mixing(MixNum).ReportFlag) {
-                auto &fromZoneHB = state.dataZoneTempPredictorCorrector->zoneHeatBalance(Mixing(MixNum).FromZone);
-                //        MixSenLoad(ZoneLoop) = MixSenLoad(ZoneLoop)+MCPM(ZoneLoop)*MAT(Mixing(MixNum)%FromZone)
-                //        H2OHtOfVap = PsyHgAirFnWTdb(ZoneAirHumRat(ZoneLoop), MAT(ZoneLoop))
```

Comment:
[src/EnergyPlus/HVACManager.cc:2687]:(style),[constVariable],Variable 'fromZoneHB' can be declared as reference to const